com.unity.sg2/Editor/GraphDeltaRegistry/FunctionDefinitions/StandardDefinitions/Input/Basic/ColorNode.cs

```diff
@@ -0,0 +1,69 @@
+using Usage = UnityEditor.ShaderGraph.GraphDelta.GraphType.Usage;
+
+namespace UnityEditor.ShaderGraph.Defs
+{
+
+    internal class ColorNode : IStandardNode
+    {
+        public static string Name => "Color";
+        public static int Version => 1;
+        public static NodeDescriptor NodeDescriptor => new(
+            Version,
+            Name,
+            functions: new FunctionDescriptor[] {
+                new(
+                    "Default",
+                    //regular color picker assumes SRGB space
+                    @"    Out = IsGammaSpace() ? Color : float4(SRGBToLinear(Color.rgb), Color.a);",
```

Comment:
It's a a precision problem that I have to use a float4 constructor here. Any suggestions on how to handle this?